docs/api-testing/vault.md

```diff
@@ -43,16 +43,28 @@ Additionally, you can import variables from Postman. See [Importing Postman Coll
 If the same variable with the same name definition exists in both the **Project Vault** and **Company Vault**, then the value in the **Project Vault** will override the one in the **Company Vault**.
 :::
 
+### Mark Variables as Sensitive
+
+You can mark a variable as sensitive if it contains a sensitive value that you do not want to appear in your tests and test reports. When you mark a variable as sensitive, the value is obfuscated anywhere the value appears: in Vault, test, tests reports, and the project dashboard's logs and metrics when the variable is used in the URL, path, or query. 
```

Comment:
```suggestion
You can mark a variable as sensitive if it contains a sensitive value that you do not want to appear in your tests and test reports. When you mark a variable as sensitive, the value is obfuscated anywhere the value appears: in Vault, tests reports, and the project dashboard's logs and metrics when the variable is used in the URL, path, or query. 
```
I'm asking to remove "test" because adding sensitive data in the composer means the value is printed in the test report, there's no way to see a variable inside the test directly